test/e2e/delete_test.go

```diff
@@ -115,7 +115,10 @@ func testWindowsNodeDeletion(t *testing.T) {
 	testCtx, err := NewTestContext()
 	require.NoError(t, err)
 
+	// set excepted node count to zero, since all Windows Nodes should be deleted in the test
+	expectedNodeCount := int32(0)
```

Comment:
nit: I'd prefer this initialized closer to its usage (around line 132)

Reply:
True and agree. I moved it up here to facilitate a more natural flow to the `MachineSets` logic. 
- [Fetch](https://github.com/openshift/windows-machine-config-operator/blob/1b7feee90c711ce964ef3be75ddf3baf2f6a12e7/test/e2e/delete_test.go#L122) the MachineSets
- [Filter](https://github.com/openshift/windows-machine-config-operator/blob/1b7feee90c711ce964ef3be75ddf3baf2f6a12e7/test/e2e/delete_test.go#L126) the MachineSets
- [Check](https://github.com/openshift/windows-machine-config-operator/blob/1b7feee90c711ce964ef3be75ddf3baf2f6a12e7/test/e2e/delete_test.go#L133) the MachineSet with the Windows Label

Happy to move it closer, in case we need to.